lib/ExpandableListItem/ExpandableListItem.less

```diff
@@ -1,11 +1,12 @@
 /* ExpandableListItem Header*/
 .moon-expandable-list-item-header {
+  padding-right: @moon-spotlight-outset + 30px;
```

Comment:
In 2.5-upkeep, this padding is comes from .moon-expandable-picker-header and this css selector is compiled after .moon-item. But, in 2.6.0-dev, the .moon-item selector comes after .moon-expandable-picker-header and overrides padding as padding: 12px;.
Hi @aarontam, @Djspaceg , how do you think on this? I think this kind of issue can be found when css sequence is reversed by the require sequence.